### PR TITLE
docs: identify language in some markdown blocks

### PR DESCRIPTION
**Pull request by erights**
I really like syntax highlighting.

See https://github.com/Agoric/agoric-sdk/pull/8415

### CODE REVIEW

File: packages/eslint-plugin/README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Endo-specific plugin
 
 You'll first need to install [ESLint](http://eslint.org):
 
-```
+```sh
```
Comment (dckc) [inline packages/eslint-plugin/README.md:9]: I think the `$` prompt makes this not `sh` code. I think it's `console` or some such.

**Review by dckc — COMMENTED** [review]
seems mostly harmless

not sure it's worth the churn, though

File: packages/eslint-plugin/README.md (hunk 1 above)
Comment (erights) [reply packages/eslint-plugin/README.md]: TIL "```console". I had no idea. Thx. Done.

**Comment by erights** [comment]
Not at all urgent. But should be trivial to review, so ping ;)

**Review by dckc — APPROVED** [review]
ok

File: packages/eslint-plugin/README.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ Endo-specific plugin
 
 You'll first need to install [ESLint](http://eslint.org):
 
-```
+```console
 $ npm i eslint --save-dev
 ```
```
[the remark this answers is not in the document]
Comment (erights) [reply packages/eslint-plugin/README.md]: Done

**Pull request merged**